requirements.txt

```diff
@@ -2,7 +2,7 @@ Flask
 Flask-SQLAlchemy
 Flask-Assets
 Flask-Mail
-Flask-WTF
+Flask-WTF==0.12
```

Comment:
The latest [version](https://flask-wtf.readthedocs.io/en/stable/changelog.html), seems to have changed the API to selectively disable CSRF token checks.